backend/twitterbot/top_contributors.py

```diff
@@ -0,0 +1,40 @@
+import collections
+from datetime import datetime, timedelta
+
+from tournesol.models.comparisons import Comparison
+from tournesol.models.poll import DEFAULT_POLL_NAME
+from tournesol.models.ratings import ContributorRating
+
+
+def get_previous_month_top_public_contributor():
+    """Return the top public contributor of the previous month."""
+
+    now = datetime.now()
+    last_month = datetime(now.year, now.month, 1) - timedelta(days=15)
+
+    public_data = ContributorRating.objects.filter(
+        is_public=True, poll__name=DEFAULT_POLL_NAME
```

Comment:
Instead of `DEFAULT_POLL_NAME` here, I'd use a variable received as a parameter by `get_previous_month_top_public_contributor`, mainly because this method could be useful anywhere in the back-end, for any poll. 

```python
def get_previous_month_top_public_contributor(poll_name: str)
```

Reply:
Cool, I wasn't sure about this one, it is indeed better as you propose.